mithril-aggregator/src/http_server/routes/statistics_routes.rs

```diff
@@ -0,0 +1,115 @@
+use mithril_common::MITHRIL_SIGNER_VERSION_HEADER;
+use std::sync::Arc;
+use warp::Filter;
+
+use crate::http_server::routes::middlewares;
+use crate::DependencyContainer;
+
+pub fn routes(
+    dependency_manager: Arc<DependencyContainer>,
+) -> impl Filter<Extract = (impl warp::Reply,), Error = warp::Rejection> + Clone {
+    post_statistics(dependency_manager)
+}
+
+/// POST /statistics/snapshot
+fn post_statistics(
+    dependency_manager: Arc<DependencyContainer>,
+) -> impl Filter<Extract = (impl warp::Reply,), Error = warp::Rejection> + Clone {
+    warp::path!("statistics" / "snapshot")
+        .and(warp::post())
+        .and(warp::header::optional::<String>(
+            MITHRIL_SIGNER_VERSION_HEADER,
```

Comment:
Shouldn't it be ```MITHRIL_API_VERSION_HEADER``` rather than the ```MITHRIL_SIGNER_VERSION_HEADER```, since mithril-client sends ```MITHRIL_API_VERSION_HEADER``` in the header of the post request?

Reply:
This is the signer version to be sent with the statistic that will be recorded, so I think this is good.